conf/modules.config

```diff
@@ -156,11 +211,11 @@
                 [
                     "${meta.id}",
                     meta.tree ? "tree: ${meta.tree}" : "",
-                    meta.args_tree ? "argstree: ${meta.args_tree}" : "",
+                    meta.args_guidetree ? "argstree: ${meta.args_guidetree}" : "",
                     meta.args_aligner ? "args: ${meta.args_aligner}" : ""
                 ].join(' ').trim()
             }
-            ext.prefix = { "${meta.id}_${meta.tree}-args-${meta.args_tree_clean}_${meta.aligner}-args-${meta.args_aligner_clean}" }
+            ext.prefix = { "${meta.id}_${meta.guidetree}-args-${meta.args_guidetree_clean}_${meta.alignment ?: meta.treealign}-args-${meta.args_alignment_clean ?: meta.args_treealign_clean}" }
```

Comment:
The name of the files is used as a tag and this info is used for the shiny and final stats. The structure doesn't change so it should work as it is.